community/kernel/src/main/java/org/neo4j/kernel/impl/api/index/IndexingService.java

```diff
@@ -201,18 +210,19 @@ public void init()
                 switch ( initialState )
                 {
                 case ONLINE:
-                    indexProxy =
-                    indexProxyCreator.createOnlineIndexProxy( indexId, descriptor, providerDescriptor );
+                    monitor.initialState( descriptor, ONLINE );
+                    indexProxy = indexProxyCreator.createOnlineIndexProxy( indexId, descriptor, providerDescriptor );
```

Comment:
Wow, so for `ONLINE` indexes previously we put a `null` `indexProxy` in the `indexMap` below?

Reply:
No, there are two changes here. Adding call to monitor and putting this on a single line. No real change here.
```
                   indexProxy =
                   indexProxyCreator.createOnlineIndexProxy( indexId, descriptor, providerDescriptor );
```